rest_framework_simplejwt/serializers.py

```diff
@@ -18,7 +18,9 @@ def __init__(self, *args, **kwargs):
 
 
 class TokenObtainSerializer(serializers.Serializer):
-    username_field = User.USERNAME_FIELD
+    user_model = get_user_model()
+
+    username_field = user_model.USERNAME_FIELD
```

Comment:
Is it really necessary to set a name "user_model" if it's only used once in L23?